.github/workflows/auto-fix.yml

```
@@ -0,0 +1,120 @@
+          PR_NUMBER=${{ github.event.issue.number }}
+          gh pr diff $PR_NUMBER --repo ${{ github.repository }} > /tmp/pr_diff.txt
+          gh pr view $PR_NUMBER --repo ${{ github.repository }} --json comments | jq -r '.comments[].body' > /tmp/pr_comments.txt
+
+      - name: Generate and Apply Fix
+        env:
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          OPENAI_BASE_URL: ${{ secrets.OPENAI_BASE_URL }}
+          OPENAI_MODEL: ${{ secrets.OPENAI_MODEL }}
+        run: |
+          python3 << 'PYTHON_EOF'
+import os, sys, json, re
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)